src/palace/manager/api/routes.py

```diff
@@ -332,6 +332,15 @@ def patron_profile():
     return app.manager.profiles.protocol()
 
 
+@library_dir_route("/patrons/me/erase_activity_history", methods=["PUT"])
```

Comment:
This isn't really erasing anything. I think `patrons/me/reset_statistics_uuid` might be a more descriptive endpoint here.